forms-flow-web/src/components/Form/EditForm/FormEdit.js

```diff
@@ -186,6 +204,38 @@ const Edit = React.memo(() => {
     };
   }, [restoredFormId]);
 
+  const fetchRestoredFormData = (restoredFormId) => {
+    if (restoredFormId) {
+      fetchFormById(restoredFormId)
+        .then((res) => {
+          if (res.data) {
+            const { data } = res;
+            dispatch(setRestoreFormData(res.data));
```

Comment:
if we are setting data variable , instead of using res.data , use 'data' itself

Reply:
updated